Description: Allows static linkage of fail-mbr
 Thanks to Matthias Klose for the patch
 .
 partclone (0.2.88-1ubuntu1) yakkety; urgency=medium
 .
   * Build statically linked fail-mbr.bin binary with -no-pie.
   * Configure with --disable-silent-rules.
Author: Matthias Klose <doko@ubuntu.com>

---
Index: partclone-0.3.6/fail-mbr/compile-mbr.sh
===================================================================
--- partclone-0.3.6.orig/fail-mbr/compile-mbr.sh
+++ partclone-0.3.6/fail-mbr/compile-mbr.sh
@@ -8,7 +8,7 @@ if dpkg-architecture -e amd64 || dpkg-ar
     #gcc -Os -Wall -W -Wshadow -Wpointer-arith -Wmissing-prototypes -Wundef -Wstrict-prototypes -g -falign-jumps=1 -falign-loops=1 -falign-functions=1 -mno-mmx -mno-sse -mno-sse2 -mno-3dnow -fno-dwarf2-cfi-asm -fno-asynchronous-unwind-tables -m32 -fno-stack-protector -mno-stack-arg-probe -Werror -Wno-trampolines -DUSE_ASCII_FAILBACK=1 -DHAVE_UNIFONT_WIDTHSPEC=1  -mrtd -mregparm=3       -fno-builtin   -m32 -Wl,--build-id=none   -nostdlib -Wl,-N,-S -Wl,-N -Wl,-Ttext,0x7C00   -o fail-mbr.image fail-mbr.o
 
     #echo "fail-mbr.bin [Done]. "
-    gcc -c fail-mbr.S -o fail-mbr.image
+    gcc -no-pie -c fail-mbr.S -o fail-mbr.image
     objcopy  -O binary  --strip-unneeded -R .note -R .comment -R .note.gnu.build-id -R .reginfo -R .rel.dyn fail-mbr.image fail-mbr.bin
 else
     echo "The architecture is not x86, so the file 'fail-mbr.bin' is not compiled"
